scripts/pi-hole/js/utils.js

```diff
@@ -239,9 +239,14 @@ function stateSaveCallback(itemName, data) {
 }
 
 function stateLoadCallback(itemName) {
+  var data;
   // Receive previous state from client's local storage area
-  var data =
-    localStorage === null ? backupStorage[itemName] || null : localStorage.getItem(itemName);
+  if (localStorage === null) {
+    var item = backupStorage[itemName];
+    data = typeof item === "undefined" ? null : item;
+  } else {
+    localStorage.getItem(itemName);
```

Comment:
I think you mean:
```js
  } else {
    data = localStorage.getItem(itemName);
  }
```

Reply:
Of course. This is what happens when I rush :(. I pushed a commit to resolve.